packages/nestjs-test-utilities/src/lib/builders/module/module.builder.ts

```diff
@@ -57,7 +57,17 @@ export class TestModuleBuilder implements ITestModuleBuilder {
       const module = this.imports[index] as NestJSModule;
       const imports = Reflect.getMetadata("imports", module);
 
-      const targetIndex = imports.findIndex((m: any) => m["name"] === target.name);
+      const targetIndex = imports.findIndex((m) => {
```

Comment:
You need to declare the type explicitly. Can you do that? Then I can approve it after the test runs. otherwise that seems to do it :)

Reply:
@jbiskur Looks like the NestJSModule type you defined above is having trouble here.. it works only if I change it to be:
```
export type NestJSModule =
  | Type<unknown>
  | DynamicModule
```

any ideas what might be the problem?  Surprised there isn't a general NestModule type from nestjs

Reply:
oh I thought you were saying to make it w.r.t the actual types for `m`
I have rewrote the loop.. 

```javascript
const targetIndex = imports.findIndex((nestModule: NestJSModule ) => {
  if (typeof nestModule === "function") {
    return nestModule["name"] === target.name
  } else if (typeof nestModule === "object") {
    const innerModule: NestJSModule = nestModule.module
    return innerModule ? innerModule["name"] === target.name : false
  } else {
    throw new NotImplementedException(
      `${typeof nestModule} is not supported by overrideModule`
    );
  }
});
```
but I'll put it back and make it any